desktop-widgets/command_divelist.cpp

```diff
@@ -264,24 +290,24 @@ void UndoTripBase::redo()
 	MainWindow::instance()->refreshDisplay();
 }
 
-void UndoTripBase::undo()
+void TripBase::undo()
 {
 	// Redo and undo do the same thing!
 	redo();
 }
 
-UndoRemoveDivesFromTrip::UndoRemoveDivesFromTrip(const QVector<dive *> &divesToRemove)
+RemoveDivesFromTrip::RemoveDivesFromTrip(const QVector<dive *> &divesToRemove)
 {
-	setText(divesToRemove.size() == 1 ? gettextFromC::tr("remove dive from trip")
-					  : gettextFromC::tr("remove %1 dives from trip").arg(divesToRemove.size()));
+	setText(divesToRemove.size() == 1 ? tr("remove dive from trip")
+					  : tr("remove %1 dives from trip").arg(divesToRemove.size()));
```

Comment:
wow, this bug was in many places... I wonder why I didn't catch it back when the code went in